src/Lucene.Net.ObjectMapping.Tests/ObjectMappingTest.cs

```diff
@@ -38,9 +38,9 @@ public void PropertyTypes()
             Assert.NotNull(doc);
             int remainingFields = 16 /* total fields */ - 1 /* null field */;
 
-            foreach (IFieldable field in doc.GetFields())
+            foreach (var field in doc.Fields)
```

Comment:
You should always use the type you want to work with, not 'var'. Please update everywhere else too.